### PR TITLE
Fixed #93 : dutch translation

### LINKED ISSUE #93: Dutch language
Sorry for not being able to make a pullrequest.
I have zipped the dutch language translation. (nl)

[tarteaucitron.nl.js.zip](https://github.com/AmauriC/tarteaucitron.js/files/2013980/tarteaucitron.nl.js.zip)

### PR DESCRIPTION
**Pull request by encreinformatique**
The Dutch translation has been added according to the issue.
Beware I cannot check the translation itself.
The nl locale has been added to the main script.

### CODE REVIEW

**Comment by AmauriC** [comment]
> Beware I cannot check the translation itself.
You can't check if it's working or if the translations are correct?

Great work :+1:

**Comment by encreinformatique** [comment]
@AmauriC I cannot check if the translation is correct. I am not a Dutch speaker.

File: lang/tarteaucitron.nl.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    "disclaimer": "Door deze services van derden toe te staan, accepteert u hun cookies en het gebruik van trackingtechnologieën die nodig zijn voor hun goede werking.",
+    "allow": "Toestaan",
+    "deny": "Weigeren",
+    "noCookie": "Deze service gebruikt geen cookie",
+    "useCookie": "Deze service kan worden geïnstalleerd",
+    "useCookieCurrent": "Deze service is geïnstalleerd",
+    "useNoCookie": "Deze service heeft geen cookies geïnstalleerd.",
+    "more": "Lees meer",
+    "source": "Bekijk de officiële website",
+    "credit": "Cookie manager mogelijk gemakt door tarteaucitron.js",
```
Comment (johandenhollander) [inline lang/tarteaucitron.nl.js]: Cookie manager mogelijk gemaakt door tarteaucitron.js

File: lang/tarteaucitron.nl.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
+/*global tarteaucitron */
+tarteaucitron.lang = {
+    "adblock": "Hallo! Deze site is transparant en laat u de services van derden kiezen die u wilt toestaan.",
+    "adblock_call": "Schakel uw adblocker uit om te beginnen met aanpassen.",
+    "reload": "Ververs de pagina",
+    
+    "alertBigScroll": "Door te blijven scrollen,",
+    "alertBigClick": "Als je doorgaat met het surfen op deze website,",
+    "alertBig": "staat je alle diensten van derden toe",
```
Comment (johandenhollander) [inline lang/tarteaucitron.nl.js]: sta je alle diensten van derden toe

**Comment by encreinformatique** [comment]
I changed both sentences marked as incorrect by @johandenhollander

**Pull request merged**